src/Interpreters/SessionLog.cpp

```diff
@@ -86,6 +86,7 @@ NamesAndTypesList SessionLogElement::getNamesAndTypes()
             AUTH_TYPE_NAME_AND_VALUE(AuthType::DOUBLE_SHA1_PASSWORD),
             AUTH_TYPE_NAME_AND_VALUE(AuthType::LDAP),
             AUTH_TYPE_NAME_AND_VALUE(AuthType::KERBEROS),
+            AUTH_TYPE_NAME_AND_VALUE(AuthType::SSL_CERTIFICATE),
```

Comment:
This should create the Enum mapping `SSL_CERTIFICATE = 6` , which is currently missing.

Not sure if `AuthenticationType::MAX) == 7` needs updating as well? There are currently 7 authentication type with SSL_CERTIFICATE being the highest mapped to 6.

Reply:
No, it would be superfluous. `AuthenticationType::MAX` is not a real authentication type, it just marks the end of the enumeration.